site/components/footer/index.js

```diff
@@ -21,107 +21,141 @@ const cx = classnames.bind(styles);
 const Footer = () => (
   <footer role="contentinfo" className={styles.footer}>
     <div className={styles.footerContainer}>
-      <InlineSVG src={mapSVG} className={styles.footerMap} />
+      <InlineSVG role="presentation" src={mapSVG} className={styles.footerMap} />
       <div className={styles.footerSections}>
         <nav role="navigation" className={cx('section', 'footerLinks')}>
-          <ul className={styles.nav}>
-            <li>
-              <a href="/">Home</a>
+          <ul role="listbox" className={styles.nav}>
+            <li role="listitem">
```

Comment:
again i think roles are not needed when underlying semantics is the same - goes for `ul`, `li` and `a` here